RevenueCatUI/CustomerCenter/Data/PurchaseInformation.swift

```diff
@@ -80,9 +85,12 @@ struct PurchaseInformation {
                         : .expired
                     return ExpirationOrRenewal(label: label, date: .date(dateString))
                 }
+                self.isLifetimeSubscription = expiresDate == nil
+
             case .nonSubscription:
                 self.explanation = .lifetime
                 self.expirationOrRenewal = nil
+                self.isLifetimeSubscription = false
```

Comment:
I think this should always be `true`, not `false`

Reply:
Technically we would also check the product type. 

By the way `subscribedProduct: StoreProduct? = nil` is not entirely correct since `subscribedProduct` doesn't have to be a subscription. We should rename it, but feel free to leave it like that since it's not part of this PR

Reply:
I'll apply this in another PR